### PR TITLE
feat: add main-page

### PR DESCRIPTION
**Pull request by NiFroPP**

### CODE REVIEW

**Review by DmitriyHnatiuk — APPROVED** [review]

File: src/assets/styles/common/Footer.module.scss [hunk 1]
```diff
@@ -0,0 +1,89 @@
+@import '/src/assets/styles/_variables';
+
+.footer {
+	height: 50px;
+
```
Comment (Polina04) [inline src/assets/styles/common/Footer.module.scss]: redundant empty line

File: src/assets/styles/common/Footer.module.scss [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	background-color: $Deep-Koamaru;
+}
+
+.container {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+
```
Comment (Polina04) [inline src/assets/styles/common/Footer.module.scss]: redundant empty line

File: src/assets/styles/common/Footer.module.scss [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+	width: 80%;
+	height: 50px;
+
```
Comment (Polina04) [inline src/assets/styles/common/Footer.module.scss]: redundant empty line

File: src/assets/styles/common/Footer.module.scss [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+	margin: 0 auto;
+}
+
+.github {
+	display: flex;
+	align-items: center;
+
```
Comment (Polina04) [inline src/assets/styles/common/Footer.module.scss]: redundant empty line

File: src/assets/styles/common/Footer.module.scss [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+	height: 50px;
+
```
Comment (Polina04) [inline src/assets/styles/common/Footer.module.scss]: redundant empty line

File: src/assets/styles/common/Header.module.scss [hunk 6]
```diff
@@ -0,0 +1,51 @@
+@import '/src/assets/styles/_variables';
```
Comment (Polina04) [inline src/assets/styles/common/Header.module.scss]: you can import this module without '_'
`@import '/src/assets/styles/variables';`

File: src/assets/styles/pages/MainPage.module.scss [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+.section {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+
+	width: 100%;
+
+	// padding: 30px 0;
```
Comment (Polina04) [inline src/assets/styles/pages/MainPage.module.scss]: are you need it? look like you can delete it

File: src/assets/styles/pages/MainPage.module.scss [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+	// border: 1px solid black;
```
Comment (Polina04) [inline src/assets/styles/pages/MainPage.module.scss]: and this

File: src/components/App/index.tsx [hunk 9]
```diff
@@ -1,10 +1,20 @@
...
+import Main from 'components/common/Main';
+
+// import styles from './index.module.scss';
+
+const App: React.FC = (): JSX.Element => (
+	<>
+		<Header />
+		{/* <<div id="content" className={styles.title}>
+			Lets start our small project
+		</div>> */}
```
Comment (Polina04) [inline src/components/App/index.tsx]: remove this comment please

File: src/components/common/Footer.tsx [hunk 10]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { RSS_HREF, GitHubItems } from 'constants/commonComponents';
+
+import s from 'assets/styles/common/Footer.module.scss';
```
Comment (Polina04) [inline src/components/common/Footer.tsx]: `s` is not a good variable name. maybe `styles`?

File: src/components/common/Header.tsx [hunk 11]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import logo from 'assets/images/header-logo.svg';
+import chat from 'assets/images/chat-icon.svg';
+
+import s from 'assets/styles/common/Header.module.scss';
```
Comment (Polina04) [inline src/components/common/Header.tsx]: maybe `styles`?

File: src/components/common/Main.tsx [hunk 12]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import s from 'assets/styles/common/Main.module.scss';
+import MainPage from 'components/pages/MainPage/MainPage';
```
Comment (Polina04) [inline src/components/common/Main.tsx]: incorrect imports order

File: src/components/common/MyButton.tsx [hunk 13]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import s from 'assets/styles/common/MyButton.module.scss';
```
Comment (Polina04) [inline src/components/common/MyButton.tsx]: s....

File: src/components/common/MyInput.tsx [hunk 14]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styles from 'assets/styles/common/MyInput.module.scss';
```
Comment (Polina04) [inline src/components/common/MyInput.tsx]: please add empty line before this line

File: src/components/common/MyInput.tsx [hunk 15]
```diff
@@ -0,0 +1,10 @@
...
+
+const MyInput: React.FC = () => {
+	const { myInput } = styles;
```
Comment (Polina04) [inline src/components/common/MyInput.tsx]: I think styles desctructuring is redundant

File: src/components/pages/MainPage/MainPage.tsx [hunk 16]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import MyButton from 'components/common/MyButton';
+import MyInput from 'components/common/MyInput';
+
+import poker from 'assets/images/MainPage/poker-planning.jpg';
+
+import s from 'assets/styles/pages/MainPage.module.scss';
```
Comment (Polina04) [inline src/components/pages/MainPage/MainPage.tsx]: please read [this](https://www.30secondsofcode.org/articles/s/javascript-naming-conventions) article about naming
 > Variable names should be self-descriptive, describing the stored value.

File: src/components/App/index.tsx [hunk 17]
```diff
@@ -1,10 +1,20 @@
...
 
-const App = (): JSX.Element => (
-	<div id="content" className={styles.title}>
-		Lets start our small project
-	</div>
+import Footer from 'components/common/Footer';
+import Header from 'components/common/Header';
+import Main from 'components/common/Main';
+
+// import styles from './index.module.scss';
```
Comment (Polina04) [inline src/components/App/index.tsx]: remove it please

**Review by Polina04 — COMMENTED** [review]
Please move every component file into folder with its name and file with styles for it should be in the sane folder. For example:
Folder name -`Button`, in this folder you have file `index.tsx` with your component code and file `index.module.scss` with styles for Button component.
`MainPage.tsx` also has to be in  the folder `MainPage` the same how I wrote above.

So, please create the same structure as you have for `App` component

**Review by DmitriyHnatiuk — APPROVED** [review]

**Review by Polina04 — APPROVED** [review]

**Pull request merged**